package/yast2-apparmor.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Aug 24 08:20:53 CDT 2018 - rgoldwyn@suse.com
+
+- Process profile change only if we have it selected (bsc#1090592)
+- 4.0.6
```

Comment:
as it is for master ( so SLE15 SP1 ) version should be 4.1.0. see https://yastgithubio.readthedocs.io/en/latest/versioning/